whirlpool/appliance.py

```diff
@@ -86,12 +86,16 @@ async def fetch_data(self):
             return False
 
         uri = f"{self._backend_selector.base_url}/api/v1/appliance/{self._said}"
-        self._data_dict = None
+        #self._data_dict = None # Clearing the _dict causes HA to complain about missing attributes 
```

Comment:
Lets leave this out of this PR, since it is not related to the websocket

Reply:
This is related to the websocket reconnect - part of the reconnect is to fetch the current data (via a fetch).  When fetch clears the dict and HA tries to poll the data it complains about missing keys and the integration dies.  Can't fix the eventsocket reconnect without this.